api/src/main/java/org/ehrbase/api/service/ValidationService.java

```diff
@@ -56,6 +57,14 @@
      */
     void check(Composition composition) throws Exception;
 
+    /**
+     * initially check if ehrstatus is valid for further processing
+     * @param ehrStatus
+     * @throws IllegalArgumentException
+     */
+    void check(EhrStatus ehrStatus);
```

Comment:
I removed the `throws Exception` here since it only throws `RuntimeException`s anyway. Alright? Would that make sense for the `check(Composition composition)`, too?